x/stake/keeper/distribute.go

```diff
@@ -136,10 +144,48 @@ func (k Keeper) DistributeInBreathBlock(ctx sdk.Context, sideChainId string) sdk
 	var toSaveValDistAddrs []types.StoredValDistAddr // mapping between validator and distribution address, to be saved
 
 	bondDenom := k.BondDenom(ctx)
+	feeFromBscToBcRatio := k.FeeFromBscToBcRatio(ctx)
+	avgFeeForBcVals := sdk.ZeroDec()
+	if sdk.IsUpgrade(sdk.BEPHHH) && sideChainId == types.MockSideChainIDForBeaconChain {
+		feeForAllBcValsCoins := k.BankKeeper.GetCoins(ctx, FeeForAllBcValsAccAddr)
+		feeForAllBcVals := feeForAllBcValsCoins.AmountOf(bondDenom)
+		avgFeeForBcVals = sdk.NewDec(feeForAllBcVals / int64(len(validators)))
```

Comment:
There is some dusk token on `FeeForAllBcValsAccAddr` because of unclear `dive`,  comfortable to keep it there, right?

Reply:
Yes.